lib/ash_admin/live/resource_live/index.html.heex

```diff
@@ -0,0 +1,41 @@
+<.live_component
```

Comment:
Personally, I prefer having the html just live in the component's `render/1`, I don't like having to jump back and forth.

Reply:
Fair enough, moved back to ex file